db/migrate_cards/data/cards/production_export

```diff
@@ -0,0 +1,51 @@
+[[home]]
```

Comment:
in data/cards I store the content for cards that get imported with the card key as file name